sirepo_bluesky/sirepo_bluesky.py

```diff
@@ -70,8 +75,9 @@ def auth(self, sim_type, sim_id):
 
     def copy_sim(self, sim_name):
         """ Create a copy of the current simulation. Returns a new instance of SirepoBluesky. """
-        assert self.sim_id
-        # simulationId, simulationType, name, folder
+        if not self.sim_id:
+            raise ValueError(f'sim_id is {repr(self.sim_id)}')
+        # simulationId, simulat)ionType, name, folder
```

Comment:
There is an extra `)` in the `simulationType` string. But I think we can remove this comment line as I don't see its value here.

Reply:
Addressed via https://github.com/NSLS-II/sirepo-bluesky/pull/89/commits/392a5b47c19cc891dc37b06d46c310832a3aa340.